tools/get_testing_version.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash -ef
 
 TESTING_VERSION=`grep -o "testing='[0-9.]\+'" mne/datasets/utils.py | cut -d \' -f 2 | sed "s/\./-/g"`
+# This can be incremented to start fresh when the cache misbehaves
+TESTING_VERSION=${TESTING_VERSION}-1
```

Comment:
bash magic, thanks @larsoner

Reply:
Really StackOverflom magic :)

https://stackoverflow.com/questions/63521430/clear-cache-in-github-actions#answer-64819132